web/server_example/main.go

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
I see the same file appearing in two places at least. Would it be possible to have a single place where this script is defined and refer to it ?